Lib/ufo2ft/filters/propagateAnchors.py

```diff
@@ -77,6 +80,22 @@ def _propagate_glyph_anchors(glyphSet, composite, processed):
                 base_components.append(component)
                 anchor_names |= {a.name for a in glyph.anchors}
 
+    if mark_components and not base_components and _is_ligature_mark(composite):
+        # The composite is a mark that is composed of other marks (E.g.
+        # "circumflexcomb_tildecomb"). Promote the mark that is positioned closest
+        # to the origin to a base.
+        try:
+            component = _component_closest_to_origin(mark_components, glyphSet)
+        except Exception as e:
```

Comment:
a bit too broad catch-all exception for my taste

Reply:
But is that a problem? The purpose here is to catch everyhting and bubble it up with information about which glyph processing failed.